pkg/controller/controller_test.go

```diff
@@ -79,9 +81,10 @@ var _ = Describe("Controller", func() {
 
 			It(test.descr, func() {
 				Expect(controller.ProcessNextItem()).To(BeTrue())
-				pod, err := fakeClient.CoreV1().Pods("").Get(test.expectPodName, metav1.GetOptions{})
-				Expect(err).NotTo(HaveOccurred())
-				Expect(pod.Name).To(Equal(test.expectPodName))
+				pod, err := getTestPod(fakeClient, test.expectPodName)
+				Expect(err).To(BeNil(), fmt.Sprintf("Expected nil err, got: %v", err))
```

Comment:
nit: generally (not always of course!) we use `Expect(err).NotTo(HaveOccurred(), "<extra-msg>")`

Reply:
ah! right, forgot about that one